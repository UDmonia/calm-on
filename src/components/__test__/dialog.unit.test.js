import React from "react";
import { render } from "@testing-library/react-native";
import Dialog from "../dialog";

describe("Dialog Test Suite", () => {
  test("Dialog snapshot matches", () => {
    const message = { text: "Dummy", style: "" };
    const { toJSON } = render(<Dialog message={message} />);
    expect(toJSON()).toMatchSnapshot();
  });
});
