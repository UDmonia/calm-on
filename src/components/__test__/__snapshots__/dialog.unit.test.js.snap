// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog Test Suite Dialog snapshot matches 1`] = `
<View
  style=""
>
  <Text
    style={
      Array [
        Object {
          "color": "black",
          "fontFamily": null,
        },
        Object {
          "color": "#FFFFFF",
          "fontFamily": "FontReg",
        },
      ]
    }
  >
    Dummy
  </Text>
</View>
`;
